en-US/firefox/whatsnew_77.lang

```diff
@@ -0,0 +1,53 @@
+## NOTE: Demo page at https://www-dev.allizom.org/firefox/77.0/whatsnew/all/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/77.0/whatsnew/all/
+
+
+# HTML page title
+;What’s new with Firefox
+What’s new with Firefox
+
+
+;Congrats! You’re using the latest version of Firefox.
+Congrats! You’re using the latest version of Firefox.
+
+
+# Main title
+;Picture-in-Picture this
```

Comment:
In this context, PiP is a verb like  in`Picture this`? Play of word, right? If yes, let's put a note.